src/Instrumentation/Event.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Class EventWithDuration.
```

Comment:
Alas, the copypasta has finally revealed itself :laughing:. 

```suggestion
 * Class Event.
```